### PR TITLE
Libevent tmux vim

### PR DESCRIPTION
**Pull request by ksorensen**
Recipes for

    libevent: Required for tmux
    tmux
    vim

The vim recipe has room for improvement for slimming down flash footprint and specifying a default vimrc configuration file. The supplied recipe builds a full featured vim (excluding gui) and installs
documentation and syntax highlighting patterns.

### CODE REVIEW

File: recipes/vim/files/vimrc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+set nobackup
+set nowb
+set noswapfile
+
+set noexpandtab
+
+set shiftwidth=8
+set softtabstop=8
+set tabstop=8
```
Comment (esben) [inline recipes/vim/files/vimrc]: :smiley:

File: recipes/vim/vim.inc [hunk 2]
```diff
@@ -0,0 +1,65 @@
+DESCRIPTION = "Vi IMproved"
+LICENSE = "Vim License (GPL-compatible)"
```
Comment (esben) [inline recipes/vim/vim.inc]: Should simply be
LICENSE = "Vim"
(search for vim on https://spdx.org/licenses/)

**Comment by esben** [comment]
LGTM :+1:

**Comment by ksorensen** [comment]
I'll fix the license, add HOMEPAGE and remove unnecessary creation of sysconfdir in do_install and resubmit the pull request.

**Comment by esben** [comment]
@ksorensen you can update a PR simply by (force) push new content to the same branch.  You don't have to close the original PR.

**Comment by ksorensen** [comment]
Good to know - will do so from now on.

**Pull request closed without merging**